src/content/products-and-services/services.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Serviços"
```

Comment:
This also sounds a list of items

Reply:
It's not a list, it's the header of the file.
For more information: https://www.npmjs.com/package/front-matter

Reply:
Thanks for the improvement, you're right, lists and json better work on this case.